cmd/flags_rpc.go

```diff
@@ -18,12 +19,12 @@ func RPCFlags() *flag.FlagSet {
 
 	flags.String(
 		addrFlag,
-		"",
+		rpc.DefaultConfig().Address,
 		"Set a custom RPC listen address (default: localhost)",
```

Comment:
Can `rpc.DefaultConfig().Address` not be used to replace `localhost` in this string?

Reply:
+1
I don't think importing config from flags is a good idea.

Reply:
Oh I meant the string `"Set a custom RPC listen address (default: localhost)"`, not `""`. Are we talking about the same thing?